runtime/src/startup/asm_arm.s

```diff
@@ -37,8 +41,8 @@ start:
 	mov r5, r0        /* Save rt_header; we use r0 for syscalls */
 	ldr r0, [r5, #0]  /* r0 = rt_header.start */
 	adds r0, #3       /* r0 = rt_header.start + 4 - 1 (for Thumb bit) */
-	cmp r0, r4
-	beq .Lset_brk     /* Skip error handling if pc correct */
+	cmp r0, r4		  /* Skip error handling if pc correct */
```

Comment:
On this line (and line 67), you have the comments aligned using tabs rather than spaces. This causes them to be misaligned in GitHub's editor. Please re-align the comments with spaces.

Reply:
Sorry about that :(, fixed.